almost-jira/src/test/java/pl/edu/pw/elka/pap/z16/almostjira/services/UserServiceUnitTest.java

```diff
@@ -0,0 +1,47 @@
+package pl.edu.pw.elka.pap.z16.almostjira.services;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import pl.edu.pw.elka.pap.z16.almostjira.exceptions.ResourceNotFoundException;
+import pl.edu.pw.elka.pap.z16.almostjira.models.User;
+import pl.edu.pw.elka.pap.z16.almostjira.repositories.UserRepository;
+
+import java.util.Date;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.BDDMockito.given;
+
+@ExtendWith(MockitoExtension.class)
+class UserServiceUnitTest {
+
+    @Mock
+    private UserRepository userRepositoryMock;
+
+    @Test
+    void shouldReturnUserById() {
+        //given
+        var user = User.builder()
```

Comment:
I think that we could add private method that will be creating an user this will simplify code when adding more tests

Reply:
Makes sense. Done✅